openraft/src/network.rs

```diff
@@ -77,6 +78,7 @@ where C: RaftTypeConfig
 {
     /// Actual type of the network handling a single connection.
     type Network: RaftNetwork<C>;
+    type ConnectError: Display;
```

Comment:
I think a tighter trait bound is needed here.

Reply:
Why isn't it `type ConnectError: Error`?